Environmental/new_crawlers/Weather forecast/anta_weather_forecast/consumer.py

```diff
@@ -0,0 +1,48 @@
+import os
+import json
+from kafka import KafkaConsumer
+from water_quality_cork.elastic import ElasticSearchClient
+
+from dotenv import load_dotenv
+
+
+"""
+In docker compose there is another exposed port (29092) for clients of Kafka
+
+"""
+
+ANTA_LAT = 36.963974
+ANTA_LON = 30.727117
+
+
+def insert_modified_data(forecast):
+    for variable, data in forecast.items():
+        doc = {"Variable": variable}
+        for time, value in data['data'].items():
+            doc['Time'] = time
+            doc['Value'] = value
+
+            doc['Unit'] = data['unit']
+            doc['Description'] = data['description']
+            doc['location'] = {
+                'lat': ANTA_LAT,
+                'lon': ANTA_LON
+            }
+            print(doc)
+            res = es.insert_doc(doc, id_=variable+time)
+            print(res)
+
+
+load_dotenv()
+
+es = ElasticSearchClient(os.environ['ELASTICSEARCH_HOST'], os.environ['ELASTICSEARCH_PORT'], "anta_env_weatherforecast_daily")
```

Comment:
maybe it is better if the client is initialized with only the host/port/username/password and the index is specified during the insertion?